sopel/modules/tell.py

```diff
@@ -32,10 +34,10 @@ def loadReminders(fn, lock):
                 line = line.decode('utf-8')
             if line:
                 try:
-                    tellee, teller, verb, timenow, msg = line.split('\t', 4)
+                    tellee, teller, verb, timenow, timestamp, msg = line.split('\t', 5)
                 except ValueError:
                     continue  # @@ hmm
-                result.setdefault(tellee, []).append((teller, verb, timenow, msg))
+                result.setdefault(tellee, []).append((teller, verb, timenow, timestamp, msg))
```

Comment:
I'm very curious what you expect to happen with old `tell` files that contain only 4 columns of data.

Aside from backward-compatibility with existing Sopel installations, let's talk about both `timenow` _and_ `timestamp` exist, since they're basically the same data in different formats.

`timenow` is defined during reminder creation as a formatted string based on the current `datetime.datetime.utcnow()` (which will be used [if no `time` is passed](https://github.com/sopel-irc/sopel/blob/ff09966ebf2d49197fd65aa55f105a38196522a2/sopel/tools/time.py#L174-L175) to `tools.time.format_time()`, as below).

`timestamp`, meanwhile, is a `datetime.datetime.timestamp()` created from `datetime.datetime.now()` (note: not `utcnow()`), cast to an int, then a string.

I honestly don't understand why there need to be two timestamps stored with the reminder, when a single UTC timestamp would suffice (and could be converted to any desired relative or absolute output format, in whatever timezone is desired, later). And the apparent lack of support for existing `tell` "database" files bothers me a lot.